packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope.ts

```diff
@@ -190,7 +190,9 @@ function createSoapBodyContent({
               ? {
                   [`${prefix}patientAddress`]: {
                     [`${prefix}value`]: patientAddresses.map(address => ({
-                      [`${prefix}streetAddressLine`]: address.line?.join(", "),
+                      ...address.line?.map(line => ({
+                        [`${prefix}streetAddressLine`]: line,
+                      })),
```

Comment:
ihe spec supports providing multiple address lines for apt numbers and secondary addresses